templates/scripts/import-cf-keypair.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+PROGNAME=$(basename "$0")
+
+function usage() {
+    echo "Usage: ${PROGNAME} [ -h | --help ] ID FILE"
+    echo
+    echo "Imports the specified CF Keypair data into AWS SecretsManager."
+    echo
+    echo "ID: CloudFront Keypair ID (Access Key ID)"
+    echo "FILE: CloudFront Keypair private key file"
+    echo
+    echo "See: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs"
+}
+
+PARSED_ARGUMENTS=$(getopt -n ${PROGNAME} -o h --long help -- "$@")
+
+while : ; do
+    case "$1" in
+        -h | --help)
+            shift
+            usage
+            exit 0
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            break
+            ;;
+    esac
+done
+
+if [ "$#" -ne 2 ]; then
+    echo "Incorrect number of arguments"
```

Comment:
How about executing the `usage` function here?

Reply:
Yeah...
I've done that previously and in using other tools I've seen more verbose output like this.
I've no strong leaning either way.
If you're in favour of the full help text, I'll do that.

Reply:
Sure, might as well. The first thing I did when I tried the script was run it with no arguments and the only feedback was that that was the wrong number of arguments. Would be nice to just have the full help text there even though the next thing most people will do is add `--help` (Okay, for most people that's probably the first thing they would do, but still.. ;) )